docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.md

```diff
@@ -0,0 +1,422 @@
+# The External DNS Helm Chart version
+export EXTERNAL_DNS_HELM_CHART_VERSION="1.13.1"
+# The Cert-Manager Helm Chart version
+export CERT_MANAGER_HELM_CHART_VERSION="1.13.2"
+# The Camunda 8 Helm Chart version
+export CAMUNDA_HELM_CHART_VERSION="8.3.3"
+```
+
+Additionally, follow the guide from either [eksctl](./eks-helm.md) or from [Terraform](./terraform-setup.md) to retrieve the following values, which will be required for subsequent steps:
+
+- EXTERNAL_DNS_IRSA_ARN
+- CERT_MANAGER_IRSA_ARN
+- DB_HOST
+- PG_USERNAME
+- PG_PASSWORD
+- DEFAULT_DB_NAME
+- REGION
+
+### DNS set up
+
+:::info
+If you don't have a domain name, you can skip to [Camunda 8](#camunda-8).
+:::
+
+#### ingress-nginx
+
+[Ingress-nginx](https://github.com/kubernetes/ingress-nginx) is an open-source Kubernetes Ingress controller that provides a way to manage external access to services within a Kubernetes cluster. It acts as a reverse proxy and load balancer, routing incoming traffic to the appropriate services based on rules defined in the Ingress resource.
+
+The following installs `ingress-nginx` in the `ingress-nginx` namespace via Helm. For more configuration options, consult the [Helm chart](https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx).
+
+```shell
+helm upgrade --install \
+  ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --version $INGRESS_HELM_CHART_VERSION \
+  --set 'controller.service.annotations.service\.beta\.kubernetes\.io\/aws-load-balancer-backend-protocol=tcp' \
+  --set 'controller.service.annotations.service\.beta\.kubernetes\.io\/aws-load-balancer-cross-zone-load-balancing-enabled=true' \
+  --set 'controller.service.annotations.service\.beta\.kubernetes\.io\/aws-load-balancer-type=nlb' \
+  --namespace ingress-nginx \
+  --create-namespace
+```
+
+#### external-dns
+
+[External-dns](https://github.com/kubernetes-sigs/external-dns) is a Kubernetes add-on that automates the management of DNS records for external resources, such as load balancers or Ingress controllers. It monitors the Kubernetes resources and dynamically updates the DNS provider with the appropriate DNS records.
+
+The following installs `external-dns` in the `external-dns` namespace via Helm. For more configuration options, consult the [Helm chart](https://github.com/kubernetes-sigs/external-dns/tree/master/charts/external-dns).
+
+Consider setting `domainFilters` via `--set` to restrict access to certain hosted zones.
+
+```shell
+helm upgrade --install \
+  external-dns external-dns \
+  --repo https://kubernetes-sigs.github.io/external-dns/ \
+  --version $EXTERNAL_DNS_HELM_CHART_VERSION \
+  --set "env[0].name=AWS_DEFAULT_REGION" \
+  --set "env[0].value=$REGION" \
+  --set txtOwnerId=external-dns \
+  --set policy=sync \
+  --set "serviceAccount.annotations.eks\.amazonaws\.com\/role-arn=$EXTERNAL_DNS_IRSA_ARN" \
+  --namespace external-dns \
+  --create-namespace
+```
+
+#### cert-manager
```

Comment:
I would suggest to link to eksctl section where IAM permissions are set as a note in this section.